core/src/main/java/com/gentics/mesh/cli/MeshImpl.java

```diff
@@ -123,11 +123,20 @@ public void run(boolean block) throws Exception {
 
 	private void setupKeystore(MeshOptions options) throws Exception {
 		String keyStorePath = options.getAuthenticationOptions().getKeystorePath();
+		if (keyStorePath == null || keyStorePath.isEmpty()) {
```

Comment:
Move this into the validation section of the MeshOptions / AuthenticationOptions.

Reply:
Done